src/libs/actions/Welcome.js

```diff
@@ -8,9 +8,11 @@ import * as Policy from './Policy';
 import ONYXKEYS from '../../ONYXKEYS';
 import NameValuePair from './NameValuePair';
 import CONST from '../../CONST';
-import createOnReadyTask from '../createOnReadyTask';
```

Comment:
It doesn't look like this is used anywhere else, so I think we can get rid of it (along with it's testing file `createOnReadyTaskTest.js`).

Do you agree @tgolen?

Reply:
Yeah, I had a few PRs that are open that refactors this, and I wasn't sure what order they were going to be merged in, so that's why I didn't include the removal in this PR. It looks like the others have been merged, so I'll update this.